WhaleSpotting/Controllers/SightingsController.cs

```diff
@@ -39,5 +39,12 @@ public IActionResult CreateSighting([FromBody] SightingRequestModel sightingRequ
                 return BadRequest(e.Message);
             }
         }
+
+        [HttpPut("/{id}/confirm")]
```

Comment:
can you get rid of the first slash here, and for the create. Basically with the / the url here is just `localhost:5001/{id}/confirm`, but we want it to be `localhost:5001/sightings/{id}/confirm`